docs/sql/ddl.txt

```diff
@@ -193,6 +193,12 @@ compute the value to partition by from existing columns in the table::
     ... ) PARTITIONED BY (partition_value);
     CREATE OK (... sec)
 
+.. note::
+
+   A generated column used as a partition by value, is not exported
+   using ``copy to`` by default. It must be explicitly added to the
+   list of columns to be included in exported rows.
```

Comment:
this changes the output format, so this advice is wrong  see https://crate.io/docs/stable/sql/reference/copy_to.html#parameters